sympy/core/numbers.py

```diff
@@ -1987,7 +1987,7 @@ def __new__(cls, expr, coeffs=Tuple(), alias=None, **args):
 
         dom = minpoly.get_domain()
 
-        if coeffs != Tuple():
+        if coeffs is not None:  # Empty list or tuple is accepted
```

Comment:
> I would help me to remember why this not just ``if coeffs:``, but I'll remove it.

Or put the default first, like `if coeffs is None:`.

(Sorry to bike-shed: others have told me to avoid too many comments in 
python and try for self-documenting code).